activestorage/lib/active_storage/engine.rb

```diff
@@ -116,7 +116,9 @@ class Engine < Rails::Engine # :nodoc:
       ActiveSupport.on_load(:active_storage_blob) do
         configs = Rails.configuration.active_storage.service_configurations ||=
           begin
+            env_config_file = Rails.root.join("config/storage/#{Rails.env}.yml")
             config_file = Rails.root.join("config/storage.yml")
+            config_file = env_config_file if File.exist?(env_config_file)
```

Comment:
Calling `#exist?` on env_config_file is more consistent with line 122 that uses `config_file.exist?`
```suggestion
            config_file = env_config_file if env_config_file.exist?
```